src/core/doh.js

```diff
@@ -13,15 +13,15 @@ import * as dnsutil from "../commons/dnsutil.js";
 import IOState from "./io-state.js";
 
 /**
- * @param {*} event
+ * @param {FetchEvent} event
```

Comment:
Node doesn't pass along a FetchEvent though...?

Reply:
yeah. This function expects it though.

this expectation came from CF workers fetch event-listener syntax (which is no longer used).

then deno had [`RequestEvent`](https://deno.land/api@v1.30.0?s=Deno.RequestEvent), with only `respondWith` method, it's own "FetchEvent".

node.js had to be poly-filled with Fetch API (no longer, i guess).

and now fastly has a it's own it's own implementation of `FetchEvent`.